docs/maintenance/Pull_Requests.mdx

```diff
@@ -24,15 +24,85 @@ Per the scales from [Contribution Tiers > Pointing](./Contributor_Tiers.mdx#poin
 - "Unusual"-categorized: require two maintainer approvals.
   These include significant refactors with cross-project and public API ramifications.
 
-## PR Flow
+## PR Review Flow
 
 :::note
 We include a set of common responses to PRs in [`.github/replies.yml`](https://github.com/typescript-eslint/typescript-eslint/blob/main/.github/replies.yml), intended to be used with the [Refined Saved Replies](https://github.com/JoshuaKGoldberg/refined-saved-replies) extension.
 Don't treat these as exact responses you must use: they're just a starting copy+paste helper.
 Please do adopt your specific responses to your personal tone and to match the thread for non-straightforward PRs.
 :::
 
-TODO: This will be filled out... soon!
+Open pull requests generally switch between two states:
+
+- Ready for review: either newly created or after [review is re-requested](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews#re-requesting-a-review)
+- Waiting for author activity: either by virtue of [being a draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests) or having the [`awaiting response` label](https://github.com/typescript-eslint/typescript-eslint/pulls?q=is%3Apr+is%3Aopen+label%3A%22awaiting+response%22)
+
+Add the `awaiting response` label to a PR whenever you post a review.
```

Comment:
I think it's worth mentioning `please fill in the template`, `blocked by another PR` and maybe something else?.. There may be a few more frequently used labels, but I'm not sure :smile:

Reply:
Right, yes! `1 approval` is a good one.